src/main.py

```diff
@@ -76,28 +77,34 @@ def run(self) -> None:
             gha_utils.append_job_summary(pull_request_body)
 
             if not self.user_config.skip_pull_request:
-                new_branch_name = f"gh-actions-update-{int(time.time())}"
+                new_branch_name = self.user_config.pull_request_branch
+                if not self.user_config.pull_request_unique:
+                    new_branch_name += f"-{int(time.time())}"
+
                 create_new_git_branch(self.env.base_branch, new_branch_name)
                 git_commit_changes(
                     self.user_config.commit_message,
                     self.user_config.git_commit_author,
                     new_branch_name,
                 )
-                pull_request_number = create_pull_request(
-                    self.user_config.pull_request_title,
-                    self.env.repository,
-                    self.env.base_branch,
-                    new_branch_name,
-                    pull_request_body,
-                    self.user_config.github_token,
-                )
-                add_pull_request_reviewers(
-                    self.env.repository,
-                    pull_request_number,
-                    self.user_config.pull_request_user_reviewers,
-                    self.user_config.pull_request_team_reviewers,
-                    self.user_config.github_token,
-                )
+                try:
+                    pull_request_number = create_pull_request(
+                        self.user_config.pull_request_title,
+                        self.env.repository,
+                        self.env.base_branch,
+                        new_branch_name,
+                        pull_request_body,
+                        self.user_config.github_token,
+                    )
+                    add_pull_request_reviewers(
```

Comment:
If `pull_request_number` is not None we can call this function

Reply:
Done.